suite-native/module-accounts-import/src/components/XpubHint.tsx

```diff
@@ -97,19 +97,21 @@ export const XpubHint = ({ networkType }: XpubScanHintSheet) => {
                 </TouchableOpacity>
             </KeyboardAvoidingView>
 
-            <BottomSheet title={title} isVisible={isVisible} onClose={handleClose}>
-                <Box paddingTop="small" justifyContent="space-between">
-                    <Video name={video} aspectRatio={1} />
-                    <VStack spacing="large" paddingTop="large">
-                        <Text color="gray600" align="center" variant="hint">
-                            {text}
-                        </Text>
-                    </VStack>
-                    <Box marginTop="extraLarge">
-                        <Button onPress={handleClose}>Got it</Button>
+            {isVisible && (
```

Comment:
Why it needs to be wrapped? You are already passing isVisible prop and doing this will probably introduce some unwanted effects like broken animation on close because component will unmount before animation can finish etc.

Reply:
The problem is that on Android, if the BottomSheet is rendered, then the screen overflows even though it's not visible. An alternative to this is to render BottomSheet and conditionally change the height of BotomSheet content (inside of the BottomSheet component) to 0 if `!isVisible`. This solution is acceptable?